zebra-consensus/src/transaction.rs

```diff
@@ -922,9 +924,21 @@ where
                         .oneshot(primitives::halo2::Item::from(orchard_shielded_data)),
                 );
 
-                // Consensus rule: The spend authorization signature
-                // MUST be a valid SpendAuthSig signature over
-                // SigHash using rk as the validating key.
+                // # Consensus
+                //
+                // > - Let SigHash be the SIGHASH transaction hash of this transaction, not
+                // >   associated with an input, as defined in § 4.10 using SIGHASH_ALL.
+                // > - The spend authorization signature MUST be a valid SpendAuthSig^{Orchard}
+                // >   signature over SigHash using rk as the validating key — i.e.
+                // >   SpendAuthSig^{Orchard}.Validate_{rk}(SigHash, spendAuthSig) = 1.
+                // >   As specified in § 5.4.7, validation of the 𝑅 component of the
+                // >   signature prohibits non-canonical encodings.
+                //
+                // https://zips.z.cash/protocol/protocol.pdf#actiondesc
+                //
+                // This is validated by the verifier, inside the `redpallas` module.
```

Comment:
Actually, why don't we change it to a reference?
```suggestion
                // This is validated by the verifier, inside the [`primitives::redpallas`] module.
```

Reply:
It's also clearer where the reader should look for the `redpallas` module.